docs/emulation.md

```diff
@@ -63,7 +63,8 @@ with a :gear:. You can get them
 - [Parallel Launcher](https://parallel-launcher.ca) - Nintendo 64 games
 - [:gear: DuckStation](https://www.duckstation.org) - PlayStation 1 games
 - [bsnes](https://github.com/bsnes-emu/bsnes) /
-  [Snes9x](https://www.snes9x.com) - Super Nintendo Entertainment System games
+- [Snes9x](https://www.snes9x.com) - Super Nintendo Entertainment System games
+- [WePlayDOS Games](https://weplaydos.games/) - Play 150+ classic dos games in browser (no-setup)
```

Comment:
You can change the description to something like "Web browser MS-DOS emulator" so that it's more consistent.

Reply:
Done